### ISSUE TITLE
Recommend installing phpunit bridge instead of phpunit

### ISSUE THREAD

**Issue by weaverryan** [symfony/symfony-docs#8808]
See https://github.com/symfony/recipes/pull/292#issuecomment-348570723

I added the 4.0 milestone, because Flex will show a warning. But really, this should likely be done in 3.3.

**Comment by weaverryan** [comment]
Also, I believe when you install the bridge with Flex, you get a `bin/phpunit` shortcut...

**Comment by ricknox** [comment]
@weaverryan Yes, due to the merged pull request at symfony/recipes/pull/293 you now get a `bin/phpunit` file. I believe that symfony/recipes/pull/292 is now null and void except for the documentation part :-)

My suggestion for the documentation part in `testing.rst`:

```markdown
.. note::

    It's recommended to use the latest stable PHPUnit version, `installed as
    PHAR`_.
```

should also contain a tip:

```markdown
.. tip::

    If you are using Flex you can simply install the `phpunit-bridge` recipe.
    Using phpunit as a dependency is discouraged.
```

Let me know your thoughts!

**Comment by javiereguiluz** [comment]
@weaverryan I prefer to make this change in 4.0 only because if we can assume that Flex is used, then everything is super simple to explain. See #8883.

**Comment by javiereguiluz** [comment]
Fixed by #8883.
